src/Components/Main.tsx

```diff
@@ -90,6 +102,38 @@ export const Main = (props: IProps) => {
     if (loadingXLSX) onShare();
   }, [loadingXLSX]);
 
+  React.useEffect(() => {
+    let _global = 0;
+    let _success = 0;
+    let _warning = 0;
+    let _failed = 0;
+
+    requests.map(item => {
+      if (item instanceof RNRequest || item instanceof NRequest) {
+        switch (getStatus(item.status)) {
+          case EnumStatus.Success:
+            _success++;
+            break;
+          case EnumStatus.Warning:
+            _warning++;
+            break;
+          default:
+            _failed++;
+            break;
+        }
+      }
+    });
+
+    _global = _success + _warning + _failed;
```

Comment:
we should check if `_global != 0` to avoid any ArithmeticException

Reply:
Done. I added a return if _global === 0